charts/streams-app/README.md

```diff
@@ -0,0 +1,110 @@
+| `nodeAffinity`           | Map to configure [pod affinities](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).                                                                                                                                 | `{}`                                       |
+| `requirePodAntiAffinity` | Whether to require replicas to be scheduled on different nodes. See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity.                                                                         | `false`                                    |
+
+### Storage
+
+| Parameter                  | Description                                                                                                                       | Default |
+|----------------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|
+| `persistence.enabled`      | Use a [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to store the state of the streams app. | `false` |
+| `persistence.size`         | Size of the persistent volume.                                                                                                    | `1Gi`   |
+| `persistence.storageClass` | Storage class to use for the persistent volume.                                                                                   |         |
+
+### Image
+
+| Parameter          | Description                                | Default      |
+|--------------------|--------------------------------------------|--------------|
+| `image`            | Docker image of the Kafka Streams app.     | `streamsApp` |
+| `imageTag`         | Docker image tag of the Kafka Streams app. | `latest`     |
+| `imagePullPolicy`  | Docker image pull policy.                  | `Always`     |
+| `imagePullSecrets` | Secrets to be used for private registries. |              |
+
+### Streams
+
+| Parameter                    | Description                                                                                                                                                                      | Default |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `streams.brokers`            | Comma separated list of Kafka brokers to connect to.                                                                                                                             |         |
+| `streams.schemaRegistryUrl`  | URL of schema registry to connect to.                                                                                                                                            |         |
```

Comment:
I like these tables as an overview. I think an additional column `required` could be useful similar to how we did it [here](https://github.com/bakdata/ci-templates/tree/main/actions/helm-gke-deploy#input-parameters), because not every parameter with an empty default is also required (e.g. `priorityClassName`)

Reply:
Besides the image basically nothing is required because everything depends on your scenario and you always need a subset of configs so I don't think it makes much sense here

Reply:
Ok, I thought the `brokers` and `schemaRegistryUrl` are always required, then it is fine